hazelcast/src/main/java/com/hazelcast/jet/impl/JobSummary.java

```diff
@@ -52,6 +53,21 @@ public JobSummary(
             @Nonnull JobStatus status,
             long submissionTime
     ) {
+        this(false, jobId, executionId, nameOrId, status, submissionTime);
+    }
+
+    /**
+     * Constructor for a light job
+     */
+    public JobSummary(
```

Comment:
We don't need to create a new overload, just add the parameter and modify every call site (they are 2 of them).